### DISCUSSION TITLE
Is there a way to start the onedrive at login?

### DISCUSSION THREAD

**Discussion by wigglethorpe** [abraunegg/onedrive#1662] [Q&A] [answered]
I always want it running when I'm logged in under Linux.  I can probably setup an auto link on login in the settings, but it will always prompt for password.  Is there any way around that?

Rocky Linux, KDE

**Comment by abraunegg** [comment, marked as answer]
@wigglethorpe 
> I always want it running when I'm logged in under Linux. I can probably setup an auto link on login in the settings, but it will always prompt for password. Is there any way around that?

This is *odd* as, once the application is authorised, it will not prompt for a password.

Unsure why you are using an auto link on login, the recommended process to start at login is to use a systemd service - see: https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#running-onedrive-as-a-system-service

**Reply by wigglethorpe** [reply]
Every time I start it from the command line, it prompts for my local password for authorization.  I have systemctl setup, (as this is a rhel based distribution) and I thought it would need to be kickstarted everytime because it wasn't running the first time I logged out and back in.  So, if it's setup as a systemctl service it will start automatically?

**Reply by abraunegg** [reply]
> Every time I start it from the command line, it prompts for my local password for authorization. I have systemctl setup, (as this is a rhel based distribution) and I thought it would need to be kickstarted everytime because it wasn't running the first time I logged out and back in. 

What is the current *exact* command you are running to start the application?
